yapapi/golem.py

```diff
@@ -113,19 +115,41 @@ def subnet_tag(self) -> Optional[str]:
         """Return the name of the subnet, or `None` if it is not set."""
         return self._engine.subnet_tag
 
-    async def __aenter__(self) -> "Golem":
+    @property
+    def operative(self) -> bool:
+        """Return True if Golem started and didn't stop"""
```

Comment:
```suggestion
        """Return `True` if this instance is currently started, `False` otherwise."""
```

Reply:
As in the previous comment, I'm not sure if "started" is a good name.